.github/workflows/stale.yml

```
@@ -0,0 +1,22 @@
+name: Close and mark stale issue
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "Oops, seems like we needed more information for this issue, please comment with more details or this issue will be closed in 7 days."
+          close-issue-message: "This issue was closed because it is missing author input."
+          stale-issue-label: "stale"
+          any-of-labels: "need-author-input"
```

We should use a shorter version more aligned with the other labels.

```suggestion
          any-of-labels: "awaiting"
```

---

I can appreciate the brevity but I think _awaiting_ is a bit ambiguous, I would suggest _awaiting-input_ but that's just two characters shorter than _need-author-input_

---

Apologies, I expressed myself poorly. I was referring to the label's simplicity (single word, no hyphens). I believe it's a no-brainer for us to understand what it means, and we're the only ones who actually need to know.

A question like `Awaiting for what?` has arguably the same weight as `Blocked by what?`. It doesn't seem like a problem to me, but if it becomes one, we could disambiguate in the label's description if necessary.

Either way, it's worth considering its use will be marginal right now, another reason I'd favor the simplicity of words and broad terms. If external participation increases and it becomes strictly necessary (as in, a _problem_), we can always revisit and rename it. 🙂

---

I take your point